configs/skip_terms.yaml

```diff
@@ -0,0 +1,9 @@
+# List of lower-cased terms indicating an article should be skipped in its entirety, by language.
+en:
```

Comment:
A comment explaining this a bit more would be great. Specifically, {{disambiguation}} looks like a variable that's set from somewhere else, and you wonder: where is it set; and is the variable name really translated into each language?

Reply:
Added in next commit. Please close this conversation if you think the comment is sufficient.